api/models/people.py

```diff
@@ -208,43 +237,63 @@ class Teacher(TeacherCommon):
 
     availability_slots = models.ManyToManyField(DayAndTimeSlot)
 
-    can_help_with_cv = models.BooleanField(default=False)
+    has_prior_teaching_experience = models.BooleanField(
+        default=False,
+        help_text="has the applicant already worked as a teacher before applying at Samantha "
+        "Smith's Group?",
+    )
+    non_teaching_help_provided = models.ManyToManyField(
+        NonTeachingHelp,
+        blank=True,
+        related_name="teachers",
+        verbose_name="Types of non-teaching help this teacher can provide to students",
+    )
+    non_teaching_help_provided_comment = models.CharField(
+        max_length=DEFAULT_CHAR_FIELD_MAX_LEN,  # prefer this to TextField for a better search
+        blank=True,
+        verbose_name="comment on additional non-teaching skills",
+        help_text="other ways in which the applicant could help the students beside listed ones",
+    )
 
-    # Peer help. When a new teacher is added, they cannot have these set to True unless they
+    # Peer support. When a new teacher is added, they cannot have these set to True unless they
     # have prior teaching experience.  However, the `.has_prior_teaching_experience` is meant
     # to stay unchanged (it describes experience before coming to Samantha Smith's Group),
     # while it's imaginable that if a teacher teaches long enough, they will be allowed to consult
     # other teachers.  So these flags being here doesn't really break the 3rd normal form.
-    can_check_syllabus = models.BooleanField(default=False)
-    can_consult_other_teachers = models.BooleanField(default=False)
-    can_give_feedback = models.BooleanField(default=False)
-    can_help_with_children_group = models.BooleanField(
+    peer_support_can_check_syllabus = models.BooleanField(
```

Comment:
Why did you decide to add `peer_support` prefix to these fields? In my opinion, it looked better without it :)

Reply:
I know, I too hate prefixes that are repeated again and again. But I want these boolean fields to be clearly separated from e.g. `can_host_speaking_club`. 

These boolean fields would be potential candidates for moving to a separate model, but it doesn't seem to be justified just yet (as opposed to `NonTeachingHelp`). We're leaving them in `Teacher`, but I want them to be separated from the rest.

Reply:
Ok, got it, let's keep it then